.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-    - "0.10"
-    - "0.12"
-    - "iojs-v2.5.0"
+    - 6
+    - "node"
```

Comment:
We want to test with Node 10 explicitly too.

Reply:
Added